src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -290,7 +290,7 @@ public class ADDR_Addresses_TDTM extends TDTM_Runnable {
         set<Id> setHHId =  mapAccIdAddr.keySet();
         list<Contact> listCon = [select Id, is_Address_Override__c, Current_Address__c, AccountId 
             from Contact where 
-                (Account.SYSTEM_AccountType__c = :CAO_Constants.HH_ACCOUNT_PROCESSOR or Account.SYSTEM_AccountType__c = :CAO_Constants.ADM_ACCOUNT_PROCESSOR)
+                (Account.RecordType.Name = :CAO_Constants.HH_ACCOUNT_PROCESSOR or Account.RecordType.Name = :CAO_Constants.ADM_ACCOUNT_PROCESSOR)
```

Comment:
Yeah, I was thinking about this earlier... This doesn't support other languages or changing the account record type label. So it's not very robust. But if we were to change it, we would need some kind of configuration. Maybe asking what is the ID of the record type that you consider Administrative, and the record type that you consider Household. It feels like a bigger issue than the one we are dealing with here, though...